### ISSUE TITLE
Add a parameter for lc0 binary

### ISSUE THREAD

**Issue by MCMic** [LeelaChessZero/lczero-client#74]
Currently lczero-client expects lc0 binary to be in the current folder.
It would be better if we could specify which binary to use or in which folder to search.

This would help packaging this tool as well, see https://aur.archlinux.org/packages/lc0-client-git for instance

**Comment by MCMic** [comment]
A parameter for networks folder would also be a big help.
(Seems to be defined in https://github.com/LeelaChessZero/lczero-client/blob/release/lc0_main.go#L611)

**Comment by madformuse** [comment]
Yeah I needed this too (well, different name same folder). Might do the job properly and create a pull request.

**Comment by madformuse** [comment]
@MCMic if you're still around could you give me any more details about how you'd like the parameters to work. I'm assuming lc0-path and data-dir (for networks and training data)?

**Comment by MCMic** [comment]
Sadly I stopped trying to train lc0 as my computer was quite bad at it, so no I do not remember.
All I can say is at that point I was trying to use the AUR package linked above.

**Comment by mooskagh** [comment]
Good idea, see also #92.
We need a volunteer for PRs. :)
